docs/src/pages/components/data-grid/getting-started/getting-started.md

```diff
@@ -200,10 +196,21 @@ The enterprise components come in two plans: Pro and Premium.
 | [Keyboard navigation](/components/data-grid/accessibility/#keyboard-navigation)           |    ✅     |                 ✅                 |                     ✅                     |
 | [Localization](/components/data-grid/localization/)                                       |    ✅     |                 ✅                 |                     ✅                     |
 
-### License key installation
+### Evaluation (trial) licenses
 
-Once you purchase a license, you'll receive a license key.
-This key should be provided to the enterprise package to remove the watermark and
+You are [free to install](https://mui.com/store/legal/mui-x-eula/#evaluation-trial-licenses) and try `DataGridPro` as long as it is not used for development of a feature intended for production.
+Please take the component for a test run, no need to contact us.
+
+## License key
+
+In addition to the license, MUI X Pro & Premium use the notion of a "license key".
+This is meant as reminder for developers to know they are not legally licensed.
+It's not a hard constraint. The hard constraint is the international IP laws.
+
+### Installation
+
+Once you purchase a license, you'll receive a license key by email.
+This key should be provided to the package to remove the watermark and
```

Comment:
```suggestion
This key should be installed to remove the watermark and
```

Reply:
This is a self-reference `### Installation -> the key should be installed`

```suggestion
This key should be provided to the package to remove the watermark and
```